src/injecting/injector.rs

```diff
@@ -131,38 +131,38 @@ impl FunctionMapper for Module {
 mod tests {
     use super::*;
     use crate::util::load_module_from_wasm;
-    use std::path::PathBuf;
+    use std::path::Path;
 
     /// WARNING: VALUES ARE FOR TEST WASM ONLY AND WILL DIFFER FOR DIFFERENT WASM BLOBS!!!
     const VALIDATE_BLOCK_GLOBAL_INDEX: usize = 1733;
     const IMPORT_SECTION_LENGTH: usize = 39;
     const MALLOC_INDEX: usize = 25;
     const WASM_INSTRUCTION_COUNT: usize = 358;
+    const WASM_PATH: &'static str = concat!(env!("CARGO_MANIFEST_DIR"), "/wasm/test.wasm");
```

Comment:
The `wasm` folder is ingored in the .gitignore file. Now the file is no present in the repo.